script.js

```diff
@@ -0,0 +1,82 @@
+const addBtn = document.getElementById('form');
+// const addBtn = document.getElementById('add');
+// Declaring varibale for the books' container
+const listContainer = document.querySelector('.list_container');
+const booksList = JSON.parse(localStorage.getItem('books')) || [];
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+    // localStorage.setItem('books', JSON.stringify(booksList));
+  }
+
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] when the book is added tothe list it is not added to the local storage even after fereshing the page, kindly fix this bug

Reply:
Sir @Munsa1, 

Can you please re-check? Because I am testing it, the 'set' & 'get' from the local storage work perfectly. 

![Capture1](https://user-images.githubusercontent.com/102612396/197953467-50a34b91-de52-48d4-b58a-f0ca2c0b6a55.JPG)